docker/install/ubuntu_install_golang.sh

```diff
@@ -0,0 +1,4 @@
+#install the necessary dependancies for golang build
+apt-get install -y golang-go
```

Comment:
in docker, we usually need always do apt-get update first, please refer to other install script.